OKTA-640211 - Update Org concept guide for Aerial content (#5779)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md

Multiple orgs allow for the complete segregation of internal and external users. In addition, apps and changes made to the internal or external org have no impact on one another. However, multiple orgs add complexity in terms of the number of environments to manage. See [Multi-tenant solutions](/docs/concepts/multi-tenancy/).

## Multiple-org management with Okta Aerial

> Okta Aerial is an administration service designed to enable multi-org management from a single, centralized account. It introduces an account layer that allows administrators to view and manage all their Okta orgs, including both production and preview environments. Okta Aerial resolves administrative challenges faced by many organizations, such as achieving multi-org visibility and securing admin access to orgs through access requests. Okta Aerial also simplifies the management of subscribed Okta products, helping you and partners reduce the total cost of managing Okta.

Reviewer: ```suggestion
Okta Aerial is an administration service that enables multi-org management from a single, centralized account. It introduces an account layer that allows admins to view and manage all their Okta orgs, including both Production and Preview environments. Okta Aerial resolves administrative challenges that are faced by many orgs, such as achieving multi-org visibility and securing admin access to orgs through access requests. Okta Aerial also simplifies how you manage subscribed Okta products, helping you and partners reduce the total cost of managing Okta.
```
